src/db.c

```diff
@@ -1716,9 +1716,9 @@ int sortGetKeys(struct redisCommand *cmd, robj **argv, int argc, getKeysResult *
     int i, j, num, *keys, found_store = 0;
     UNUSED(cmd);
 
-    num = 0;
     keys = getKeysPrepareResult(result, 2); /* Alloc 2 places for the worst case. */
-    keys[num++] = 1; /* <sort-key> is always present. */
+    keys[0] = 1; /* <sort-key> is always present. */
```

Comment:
We generally prefer to avoid this type of small refactor since it complicates git history.